base/grafana/deploy-grafana.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -exuo pipefail
+command -v envsubst
+
+FILE_ROOT=${BASH_SOURCE%/*}
+NAMESPACE=${NAMESPACE:=tools}
+
+export FILE_ROOT NAMESPACE
+
+# make sure user-workloads are enabled
+USER_WORKLOADS=$(oc get cm/cluster-monitoring-config -n openshift-monitoring -ojsonpath='{.data.config\.yaml}')
+
+if ! [ "$USER_WORKLOADS" = "enableUserWorkload: true" ]; then
+    echo "User Workload is not enabled"
+    exit ;
+fi
+
+envsubst < "${FILE_ROOT}"/operator-group.yaml | oc apply -n "${NAMESPACE}" -f -
+oc apply -f "${FILE_ROOT}"/subscription.yaml -n ${NAMESPACE}
+
+oc wait -n "${NAMESPACE}" installplan "$(oc get -n "${NAMESPACE}" subscription grafana-operator -o=jsonpath='{.status.installPlanRef.name}')" --for=condition=Installed --timeout=120s
```

Comment:
@averevki This returns:
```
++ oc get -n tools subscription grafana-operator '-o=jsonpath={.status.installPlanRef.name}'
+ oc wait -n tools installplan '' --for=condition=Installed --timeout=120s
error: resource name may not be empty
```

If I rerun command, this wait works.

Reply:
You need to wait first for the reference to appear in the subscription. Return the wait you removed above it `oc wait -n "${NAMESPACE}" --for=jsonpath=status.installPlanRef.name subscription grafana-operator --timeout=120s`